packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.4
+
+* Introduces new ReplacementMode for Android's billing client
```

Comment:
This should also add that `ProrationMode` is being deprecated.

Reply:
I made the requested changes